Includes/ftpConnection.cpp

```diff
@@ -78,28 +80,39 @@ void ftpConnection::sendStdString(int fd, std::string const& s, int flags = 0) {
 
 ftpConnection::ftpConnection(int fd, ftpServer* s) :
   _fd(fd), server(s) {
+  buf = (char*)malloc(FTP_CMD_BUFFER_SIZE);
```

Comment:
Unchecked malloc? Is it safe to assume `buf != NULL`?

Reply:
Probably not safe, no. I've added an `assert()`. Thanks!

Reply:
Not sure if an assert is a good solution. How does nevolutionX behave if it runs out of memory? Does the whole application crash then? Does this break xboxes permanently when it is used as the default dashboard. (ofc, its highly unlikely but still...)

Reply:
You're right (as so often before). Now there's a check in the beginning of `update()` instead.